### PR TITLE
Tool/Workflow Sidebar Updates - SEAB-5165

### PR DESCRIPTION
**Pull request by KimberleyChong**
**Description**
Updated tool and workflow sidebars to match the zeplin mock-ups.

### Workflow Sidebar ([Zeplin](https://app.zeplin.io/project/5fc690a2717ee9144814f190/screen/60219bda226b07286947fbeb))
| Before  | After |
|:---:|:---:|
|  |  | 

**Additional Launch with Buttons:**
|  |  |  |
|:---:|:---:|:---:|

### Tool Sidebar ([Zeplin](https://app.zeplin.io/project/5fc690a2717ee9144814f190/screen/6021aea03d1a3115fd55b473))
| Before  | After |
|:---:|:---:|
|  |  | 

**Additional Registry Buttons**
|  |  |  |
|:---:|:---:|:---:|

**Notable changes:**
- `workflow.component` and `container.component` shared most of their styles so I created a shared stylesheet for them in `app/shared/styles/workflow-container.component.scss` and deleted their previous stylesheets 
- The light purple highlighting style from the search sidebar appears to be pretty common as it is also used for these sidebar so I removed it from `search.component.scss` and added a global style for it in `styles.scss`

https://github.com/dockstore/dockstore-ui2/blob/a4a2000317e5556838d4ef327e25f2539cc85de2/src/styles.scss#L1384-L1388

- I added a class `underline-none` to the `no-underline:hover` style since `no-underline` has another style declared above which dictates text colour, this class can be used when you want no underline but don't need the previous style

https://github.com/dockstore/dockstore-ui2/blob/a4a2000317e5556838d4ef327e25f2539cc85de2/src/styles.scss#L365-L376
- Removed the `old-launch-with-button` styles and class
- Added a handful of new icons for the new source repositories buttons

**Review Instructions**
Check it out and see if it looks good.

**Issue**
[SEAB-5165
](https://ucsc-cgl.atlassian.net/browse/SEAB-5165)
**Security**
If there are any concerns that require extra attention from the security team, highlight them here.

Please make sure that you've checked the following before submitting your pull request. Thanks!

- [x] Check that your code compiles by running `npm run build`
- [x] Ensure that the PR targets the correct branch. Check the milestone or fix version of the ticket.
- [x] If this is the first time you're submitting a PR or even if you just need a refresher, consider reviewing our [style guide](https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Opinionated-Style-Guide#pr-checklist)
- [x] Do not bypass Angular sanitization (bypassSecurityTrustHtml, etc.), or justify why you need to do so
- [x] If displaying markdown, use the `markdown-wrapper` component, which does extra sanitization
- [x] Do not use cookies, although this may change in the future
- [x] Run `npm audit` and ensure you are not introducing new vulnerabilities
- [x] Do due diligence on new 3rd party libraries, checking for CVEs
- [x] Don't allow user-uploaded images to be served from the Dockstore domain
- [x] If this PR is for a user-facing feature, create and link a documentation ticket for this feature (usually in the same milestone as the linked issue). Style points if you create a documentation PR directly and link that instead.
- [x] Check whether this PR disables tests. If it legitimately needs to disable a test, create a new ticket to re-enable it in a specific milestone.

### CODE REVIEW

**Review by coverbeck — APPROVED** [review]

File: src/app/container/container.component.html [hunk 1]
```diff
@@ -300,39 +300,42 @@ <h3>
...
+                class="top-down-padding my-1 px-2"
+                [ngClass]="{ 'selected-item weight-bold': selectedVersion?.name === sortedVersion?.name }"
+              >
+                <div>
+                  <a (click)="onSelectedVersionChange(sortedVersion)" class="panel-body-anchor">{{ sortedVersion.name }}</a
+                  >&nbsp;
+                  <span *ngIf="sortedVersion?.verified">
+                    <mat-chip matTooltip="Verified" class="bubble">
+                      <a [href]="verifiedLink" class="verified-check">
+                        <mat-icon class="verified-icon">done</mat-icon>
```
[the remark this answers is not in the document]
Comment (KimberleyChong) [reply src/app/container/container.component.html:320]: Oh the `done` is to add the `mat-icon` checkmark for the verified versions.  Removing it removes the icon 

| image | image |
|:---:|:---:|

File: src/app/workflow/workflow.component.html [hunk 2]
```diff
@@ -402,43 +402,54 @@ <h3>
...
+                class="top-down-padding my-1 px-2"
+                [ngClass]="{ 'selected-item weight-bold': selectedVersion?.name === sortedVersion?.name }"
+              >
+                <div>
+                  <a (click)="onSelectedVersionChange(sortedVersion)" class="panel-body-anchor">{{ sortedVersion.name }}</a
+                  >&nbsp;
+                  <span *ngIf="sortedVersion?.verified">
+                    <mat-chip matTooltip="Verified" class="bubble">
+                      <a [href]="verifiedLink" class="verified-check">
+                        <mat-icon class="verified-icon">done</mat-icon>
```
[the remark this answers is not in the document]
Comment (KimberleyChong) [reply src/app/workflow/workflow.component.html:422]: Same as previous response ^

File: src/app/container/container.component.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (coverbeck) [reply src/app/container/container.component.html:320]: Whoops, I misread the mat-icon documentation. I initially didn't see any examples with content.

Sorry for the confusion.

**Comment by denis-yuen** [comment]
> * The light purple highlighting style from the search sidebar appears to be pretty common as it is also used for these sidebar so I removed it from `search.component.scss` and added a global style for it in `styles.scss`

The highlighting in general as opposed to the old subtle bolding is a nice touch

**Comment by denis-yuen** [comment]
> 

The alignment on the "Add to my collection" button is a bit strange. 
Can you compare it with the buttons on the headers of https://qa.dockstore.org/dashboard?newDashboard=  (register a tool, service, organization, etc.,)

**Review by denis-yuen — APPROVED** [review]
One alignment/spacing issue to look into, looks great in general!

**Comment by KimberleyChong** [comment]
> The alignment on the "Add to my collection" button is a bit strange.

Yeah I thought that too, I actually purposely _decreased_ the padding below the button to match the mocks:

image

But I'll revert it to match the other spacing.  Going to wait for any other feedback before pushing the changes.

**Review by svonworl — APPROVED** [review]

**Review by kathy-t — APPROVED** [review]
Nice work, thanks for moving common styles out to shared/global sheets

File: src/app/entry/current-collections/current-collections.component.html [hunk 3]
```diff
@@ -14,11 +14,25 @@
...
+    <div *ngIf="isLoggedIn$ | async">
+      <button
+        mat-button
+        class="private-btn small-btn-structure"
+        matTooltip="Add the entry to a collection"
+        (click)="addEntryToCollection()"
+        id="addToolToCollectionButton"
+      >
+        <img src="../../../assets/svg/icons-plus.svg" class="pr-2 pb-1" alt="Add to my collection" />
+        <span fxHide.lt-lg>Add to my collection</span>
```
[the remark this answers is not in the document]
Comment (KimberleyChong) [reply src/app/entry/current-collections/current-collections.component.html:28]: Ou thanks!

**Pull request merged**